mesh/mesh_readers.cpp

```diff
@@ -862,30 +866,39 @@ void Mesh::ReadVTKMesh(std::istream &input, int &curved, int &read_gf,
       cell_types.Load(ncells, input);
```

Comment:
Not strictly related to this PR, but the `CELL_TYPES` are actually mandatory, so we should error if we don't find them:
```diff
diff --git a/mesh/mesh_readers.cpp b/mesh/mesh_readers.cpp
index b67105aee..cc9ec80e4 100644
--- a/mesh/mesh_readers.cpp
+++ b/mesh/mesh_readers.cpp
@@ -860,11 +860,9 @@ void Mesh::ReadVTKMesh(std::istream &input, int &curved, int &read_gf,
    input >> ws >> buff;
    Array<int> cell_types;
    int ncells;
-   if (buff == "CELL_TYPES")
-   {
-      input >> ncells;
-      cell_types.Load(ncells, input);
-   }
+   MFEM_VERIFY(buff == "CELL_TYPES", "CELL_TYPES not provided in VTK mesh.")
+   input >> ncells;
+   cell_types.Load(ncells, input);
 
    while ((input.good()) && (buff != "CELL_DATA"))
    {
```